solution/ui/prototype/src/views/Part.vue

```diff
@@ -113,17 +141,20 @@ import {
     getPart,
     getSupplementalContentCountForPart,
     getPartTOC,
-    getSectionsForPart,
 } from "@/utilities/api";
 
 import _isEmpty from "lodash/isEmpty";
 import _throttle from "lodash/throttle";
 import _isUndefined from "lodash/isUndefined";
+import LeftNav from "@/components/part/LeftNav";
```

Comment:
this should be imported up with the rest of the components, ideally on line 131 to retain alphabetical ordering